packages/drawer/src/types.tsx

```diff
@@ -0,0 +1,117 @@
+import { StyleProp, ViewStyle, TextStyle } from 'react-native';
```

Comment:
everything for this point is copied right?

Reply:
yeah, the only thing I have changed are types + replaced `onDrawerX` callbacks with events.

Reply:
oh, I also added `DrawerRouter` (second file from last)